packages/core/src/transports/ankr/adapters.ts

```diff
@@ -0,0 +1,29 @@
+import type { UTXOTransaction } from '../../types/transaction.js'
+import type { AnkrTransaction } from './ankr.types.js'
+
+export const AnkrUTXOTxnAdapter = (
```

Comment:
Why do we call this an adapter and use PascalCase? I see we use this only for mapping, so let's use one of these:

1. `toAnkrTransaction`
2. `convertToAnkrTransaction`
3. `transformToAnkrTransaction`

Also, maybe we should rename this to `transformers.ts` or `converters.ts`, adapters in the context of the wallet library might be confusing.